audb/core/load.py

```diff
@@ -977,7 +977,7 @@ def load(
             db_is_complete = _database_is_complete(db)
 
             # load attachments
-            if not db_is_complete:
+            if not db_is_complete and not only_metadata:
```

Comment:
Now that we can load a database without attachments we need to extend `_database_check_complete()` and return only True if also all attachments are in the cache.

Reply:
Theoretically a user could load all media with `load_media()` :)